packages/nodes-base/nodes/HaloPSA/descriptions/ClientDescription.ts

```diff
@@ -0,0 +1,74 @@
+import { INodeProperties } from 'n8n-workflow';
+
+export const clientDescription: INodeProperties[] = [
+	{
+		displayName: 'Name:',
```

Comment:
Name is only required when creating, not when updating. So, when updating, move to a collection.